codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -112,7 +112,7 @@
 
     private HttpMessage message;
     private long chunkSize;
-    private long contentLength = Long.MIN_VALUE;
+    protected long contentLength = Long.MIN_VALUE;
```

Comment:
@Bennett-Lynch why change this to protected ?

Reply:
I'm fine to revert this, but the motivation was to allow someone who is overriding `handleTransferEncodingChunkedWithContentLength()` to be able to manipulate this variable, if needed. I will revert this unless you think we should keep it.